main.tf

```diff
@@ -131,3 +207,18 @@ data "cloudflare_zones" "domain" {
     status      = "active"
   }
 }
+
+
+/*
+ * AWS data
+ */
+
+data "aws_caller_identity" "this" {}
+
+data "aws_region" "current" {}
+
+locals {
```

Comment:
Don't you want all the locals in a single block (see the first lines in this file)?

Reply:
I don't always do that. Sometimes I like to have them close to where they're used. What's your preference?

Reply:
I think it's better to have them together. That at least helps with PRs to avoid suspecting someone of referring to a local without adding it to the locals. :-)